src/mscorlib/src/System/Collections/Concurrent/ConcurrentQueue.cs

```diff
@@ -1109,12 +1109,23 @@ internal struct Slot
         }
     }
 
+    /// <summary>A placeholder class for common padding constants and eventually routines.</summary>
+    internal static class PaddedHelper
```

Comment:
@sdmaclea the location sounds reasonable to me, but @jkotas knows the structure of mscorlib much better, so he should provide a better feedback on that than me.